public/less/generic.less

```diff
@@ -24,84 +24,41 @@
 }
 
 // -- Bootstrap styling ---
-
-.btn, .list-group-item {
-	position: relative;
-	& > .progress {
-		position: absolute;
-		left: 0px;
-		top: 0px;
-		width: 100%;
-		height: 100%;
-		background: transparent;
-		pointer-events:none;
-		.progress-bar {
-			pointer-events:none;
-		}
-	}
-}
-.btn-group {
-	.btn > .progress {
-		border-radius: 0px;
-		.progress-bar {
-			border-radius: 0px;
-		}
-	}
-}
-
 .panel {
-	border: 0px;
+  border: none;
 }
 
 .form-control {
-	border: 0px;
-	color: rgba(255, 255, 255, 0.8);
```

Comment:
This breaks many input text colors:
![image](https://user-images.githubusercontent.com/4009570/80533986-c7507b80-899e-11ea-8a64-eb936dd9329f.png)

![image](https://user-images.githubusercontent.com/4009570/80534211-2adaa900-899f-11ea-99ad-195faebb57b3.png)

Reply:
Good catch.

I added a line to `variables.less` to set the input color and forgot to include that file.
Should be fixed with the latest commit.